download/index.md

```diff
@@ -17,7 +17,7 @@ Lean 4 is under development. There are no official releases yet.
 
 # Lean 3
 
-These are the official releases of Lean 3. For the extended community version of Lean 3, please see [leanprover-community/lean](https://github.com/leanprover-community/lean/).
+Below are the frozen official releases of Lean 3. For the extended community version of Lean 3, please see the [community website](https://leanprover-community.github.io/).
```

Comment:
"frozen releases" is a bit weird since releases are usually immutable; it's development on new releases that's frozen. I think it's fine without "frozen", but feel free to suggest alternatives.

Reply:
Yes, it sounds weird and probably doesn't really convey what I meant to people who don't already know about it. The easy way is indeed to remove that word.